src/hooks/usePushNotifications.tsx

```diff
@@ -1,9 +1,9 @@
 import React, { createContext, useContext, useEffect } from 'react';
 import {
-  PushNotificationsConfig,
   registerDeviceToken,
   requestNotificationsPermissions,
-} from '../../src/common';
+} from '../../src/common/Notifications';
```

Comment:
This reduces the imported dependencies when importing this hook and makes unit testing more concise.

Reply:
☝️ we really need _always_ do this.  If you see other imports from an index file like `/src/common`, it needs to be updated semi-soon so we're not over-importing what's needed for each file/module.

Reply:
I will follow up with another PR to fix these across the entire project.